src/Input.js

```diff
@@ -72,11 +73,16 @@ class Input extends React.Component {
       }
     }
```

Comment:
I'd do a regex check to see if it is a word/not a number to trigger this logic. It works with the custom sizes better.